packages/engine/src/interaction/behaviors/calcBoundingBox.ts

```diff
@@ -0,0 +1,88 @@
+import { Object3D, Ray, Raycaster, Vector3, Vector2, Mesh, Frustum, Matrix4, Box3, Scene } from "three";
+import { Behavior } from "../../common/interfaces/Behavior";
+import { Entity } from "../../ecs/classes/Entity";
+import { InteractBehaviorArguments } from "../types";
+import { getComponent, getMutableComponent, hasComponent } from "../../ecs/functions/EntityFunctions";
+
+import { Object3DComponent } from "../../common/components/Object3DComponent";
+import { Interactive } from "../components/Interactive";
+import { BoundingBox } from "../components/BoundingBox";
+import { TransformComponent } from '@xr3ngine/engine/src/transform/components/TransformComponent';
+import { FollowCameraComponent } from "../../camera/components/FollowCameraComponent";
+import { Interacts } from "../components/Interacts";
+import { Engine } from "../../ecs/classes/Engine";
+
+/**
+ * Checks if entity can interact with any of entities listed in 'interactive' array, checking distance, guards and raycast
+ * @param entity
+ * @param interactive
+ * @param delta
+ */
+
+ function searchModelChildrenByName(entity, name) {
+  // console.warn(entity);
+
+   let screne = getComponent(entity, Object3DComponent).value
+  // console.warn(screne);
+
+   let m
+   screne.children[0].traverse(value => {
```

Comment:
maybe you can use this method instead:
https://threejs.org/docs/#api/en/core/Object3D.getObjectByName